project-api/test-artifacts/idl-import-simple-compiler-plugin/src/main/java/io/idl/plugins/simpleclient/SimpleClientGeneratorPlugin.java

```diff
@@ -1,240 +0,0 @@
-/*
```

Comment:
Are the test files removed? E.g., IDLClientCompilerTests, IDLClientGenPluginTests?

Reply:
These files are already not in the branch.